imports/wasi_snapshot_preview1/fs.go

```diff
@@ -831,26 +831,26 @@ func fdReaddirFn(_ context.Context, mod api.Module, params []uint64) Errno {
 
 	// The host keeps state for any unread entries from the prior call because
 	// we cannot seek to a previous directory position. Collect these entries.
-	entries, errno := lastDirEntries(dir, cookie)
+	dirents, errno := lastDirents(dir, cookie)
```

Comment:
real nit, but I guess `Dirent` instead of `DirEnt` and `Readdir` instead of `ReadDir` is for consistency with WASI (libc) defs, right?

Reply:
indeed posix mostly, but also there's an odd difference in go between ReadDir and Readdir, and this is closer.

Reply:
added a comment